jekyll/_docs/enterprise/faq.md

```diff
@@ -121,6 +121,16 @@ $ # To get private ip address
 $ ifconfig eth0 | grep "inet addr"
           inet addr:10.0.0.235  Bcast:10.0.0.255  Mask:255.255.255.0
 ```
+
+#### How can I change my passphrase?
+
+First, you need to change your passphrase from the system console (services box port 8800) settings page and restart your app.
+
+Then, you need to update `CIRCLE_SECRET_PASSPHRASE` in the init script that you are using to add builder boxes to your fleet. If you are
+using AWS ASG to manage your builder boxes, you can update the environment variable in the [ASG configuration]({{ site.baseurl }}/enterprise/faq/#can-i-use-aws-auto-scaling-groups-to-manage-my-builder-boxes).
```

Comment:
@GERey That FAQ doc kim links to needs to be updated, since it lists two different init-builder scripts. I also think this section: https://circleci.com/docs/enterprise/faq/#my-github-enterprise-installation-doesnt-use-ssl-or-uses-a-self-signed-certificate should just be removed and added to the advanced config section you're working on.